src/Symfony/Component/Security/Http/RememberMe/RememberMeDetails.php

```diff
@@ -148,6 +92,6 @@ public function getValue(): string
     public function toString(): string
     {
         // $userIdentifier is encoded because it might contain COOKIE_DELIMITER, we assume other values don't
-        return implode(self::COOKIE_DELIMITER, [strtr($this->userFqcn ?? '', '\\', '.'), strtr(base64_encode($this->userIdentifier), '+/=', '-_~'), $this->expires, $this->value]);
+        return implode(self::COOKIE_DELIMITER, [strtr(base64_encode($this->userIdentifier), '+/=', '-_~'), $this->expires, $this->value]);
```

Comment:
I this we should at least preserve the current layout for easier BC/FC
```suggestion
        return implode(self::COOKIE_DELIMITER, ['', strtr(base64_encode($this->userIdentifier), '+/=', '-_~'), $this->expires, $this->value]);
```